tests/lib/cli-engine.js

```diff
@@ -446,7 +446,7 @@ describe("CLIEngine", function() {
 
             var report = engine.executeOnFiles([getFixturePath("formatters")]);
 
-            assert.equal(report.results.length, 2);
+            assert.equal(report.results.length, 3);
```

Comment:
The test is running against all files in the formatters fixtures directory. I had to add another fixture (formatters/broken.js) which turns up in the generated report:

```js
console.log(report)
// { results: 
//    [ { filePath: '/tmp/eslint/fixtures/formatters/broken.js',
//        messages: [],
//        errorCount: 0,
//        warningCount: 0 },
//      { filePath: '/tmp/eslint/fixtures/formatters/simple.js',
//        messages: [],
//        errorCount: 0,
//        warningCount: 0 },
//      { filePath: '/tmp/eslint/fixtures/formatters/test/simple.js',
//        messages: [],
//        errorCount: 0,
//        warningCount: 0 } ],
//   errorCount: 0,
//   warningCount: 0 }
```

I probably should add the missing asserts for the 3rd item in the array to fix the rest of the test up.